src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/xlf/LSPFrameworkResources.cs.xlf

```diff
@@ -1,42 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These were no longer used.  the strings are in the protocol project.  Additionally there was no resx file for any of these.